build/build.go

```diff
@@ -1,32 +1,52 @@
 // Package build is for building source into a package
 package build
 
-// Build is an interface for building packages
-type Build interface {
+// Builder is an interface for building packages
+type Builder interface {
 	// Package builds a package
-	Package(name string, src *Source) (*Package, error)
-	// Remove removes the package
-	Remove(*Package) error
```

Comment:
If you have no way of removing docker images then there's nothing that enables a cleanup process, same for tarballs and other sources. Its important that we have a way of actually removing them.

Reply:
I'm not sure if this falls outside the responsibility of the builder. I'll have a think about this and get back to you.

Reply:
My interface had Remove. Please add back remove.